src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/SourceWriter.cs

```diff
@@ -463,52 +458,38 @@ private void WriteSetDiagnostics(Node node)
 
         private void WriteGreenAcceptMethods(Node node)
         {
-            //WriteLine();
-            //WriteLine("    public override TResult Accept<TArgument, TResult>(SyntaxVisitor<TArgument, TResult> visitor, TArgument argument)");
-            //WriteLine("    {");
-            //WriteLine("        return visitor.Visit{0}(this, argument);", StripPost(node.Name, "Syntax"));
-            //WriteLine("    }");
-            WriteLine();
-            WriteLine("    public override TResult Accept<TResult>(CSharpSyntaxVisitor<TResult> visitor) => visitor.Visit{0}(this);", StripPost(node.Name, "Syntax"));
             WriteLine();
-            WriteLine("    public override void Accept(CSharpSyntaxVisitor visitor) => visitor.Visit{0}(this);", StripPost(node.Name, "Syntax"));
+            WriteLine("public override void Accept(CSharpSyntaxVisitor visitor) => visitor.Visit{0}(this);", StripPost(node.Name, "Syntax"));
+            WriteLine("public override TResult Accept<TResult>(CSharpSyntaxVisitor<TResult> visitor) => visitor.Visit{0}(this);", StripPost(node.Name, "Syntax"));
         }
 
         private void WriteGreenVisitors()
         {
-            //WriteGreenVisitor(true, true);
-            //WriteLine();
-            WriteGreenVisitor(false, true);
-            WriteLine();
-            WriteGreenVisitor(false, false);
+            WriteGreenVisitor(withResult: true);
+            WriteGreenVisitor(withResult: false);
```

Comment:
both calls passed 'false' for the first argument.  removed hte argument and simplified the impl since htis is always false.